src/page/sign-in-step.page.ts

```diff
@@ -0,0 +1,15 @@
+import { $, ElementFinder } from 'protractor';
+
+export class SignInStepPage {
+  private credentials: ElementFinder;
+
+  constructor () {
+    this.credentials = $('#SubmitLogin > span');
+  }
+
+  public async goToSignIn(): Promise<void> {
+    await $('#email').sendKeys('aperdomobo@gmail.com');
+    await $('#passwd').sendKeys('WorkshopProtractor');
```

Comment:
move as method params instead of hardcoding the user and the password

Reply:
Agree, please add two params to this method, one for email and the other one for password, and from the test class, you can pass the values. In this way, this page object will not be coupled to a specific user and password